$NetBSD: patch-Makefile.am,v 1.5 2025/06/02 08:07:34 wiz Exp $

Actually escape the "#".

If "\x23d" is used it can get substituted with "=" and that will mess up
src/version.h.

Needed due to bin/59453: sed 's/^/\x23def/' output '=ef' instead of '#def'

--- Makefile.am.orig	2025-06-01 18:07:55.412230053 +0000
+++ Makefile.am
@@ -99,7 +99,7 @@ endif
 # Remake the version.h header file if, and only if, the git ID has changed
 .PHONY: .FORCE
 .FORCE:
-generate_ver = ver="`{ $(srcdir)/scripts/version || echo '$(VERSION)' ; } | sed 's/.*/\x23define JQ_VERSION \"&\"/'`"
+generate_ver = ver="`{ $(srcdir)/scripts/version || echo '$(VERSION)' ; } | sed 's/.*/\#define JQ_VERSION \"&\"/'`"
 .remake-version-h: .FORCE
 	@ $(generate_ver); test "x`cat src/version.h 2>/dev/null`" = "x$$ver" || touch .remake-version-h
 src/version.h: .remake-version-h
